cmd/crc/cmd/setup.go

```diff
@@ -21,8 +21,8 @@ func init() {
 
 var setupCmd = &cobra.Command{
 	Use:   "setup",
-	Short: "setup hypervisor",
-	Long:  "setup hypervisor to run the cluster",
+	Short: "Setup hypervisor",
```

Comment:
`Setup requirements` as the hypervisor is JUST a small part of this.

Reply:
Note that "set up" (note the space) is the action of setting something up.

Reply:
Agreed on "set up". But would "prerequisites" sound better than "requirements"? For software engineers, "requirements" has a meaning different from what we're trying to say here.

Reply:
`Set up prerequisites` would work for me too.

Reply:
Fixed.